scripts/popolo/__init__.py

```diff
@@ -117,3 +117,16 @@ def verify(self):
     def dump(self):
         self.verify()
         json.dump(self.json, open(JSON, 'w'), indent=2, sort_keys=True)
+
+
+class PopoloPlus(Popolo):
```

Comment:
Was there a reason this wasn't included in the main class? Would seem to make sense there.